browser/brave_rewards/android/brave_rewards_native_worker.cc

```diff
@@ -700,6 +700,9 @@ void BraveRewardsNativeWorker::GetExternalWallet(
 void BraveRewardsNativeWorker::OnGetExternalWallet(
     const ledger::type::Result result,
     ledger::type::UpholdWalletPtr wallet) {
+  if(!wallet) {
+    return;
+  }
```

Comment:
Curious if we know when this happens, and when it does happen, are we handling the error correctly? I agree crashing is not right but wonder if other bases are covered too?

Reply:
@bbondy i was curious about the same thing. Adding a null check would definitely avoid the crash but i think we should resolve it from the shared lib side.

Reply:
`GetExternalWallet` in shared lib will return null in some cases, so this is expected

Reply:
Which are the cases when it returns null, and if it returns null what should the handling be?  Is there a way for QA to trigger a null wallet here?

Reply:
Drive by so I get notified on this pull, and a style request: please use consistent `if (` style (space after `if`).